<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SymbolInfoDialogForm</class>
 <widget class="QDialog" name="SymbolInfoDialogForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Symbol Information</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="m_idLabel">
         <property name="text">
          <string>Identifier:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="m_idLineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="m_nameLabel">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="m_nameLineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="m_authorLabel">
         <property name="text">
          <string>Author:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="m_authorLineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="m_categoryLabel">
         <property name="text">
          <string>Category:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="m_categoryLineEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="m_tagsLabel">
         <property name="text">
          <string>Tags:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="m_tagsLineEdit"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="m_descriptionLabel">
         <property name="text">
          <string>Description</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTextEdit" name="m_descriptionTextEdit">
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="tabChangesFocus">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="m_okPushButton">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="m_cancelPushButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_okPushButton</sender>
   <signal>clicked()</signal>
   <receiver>SymbolInfoDialogForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>70</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_cancelPushButton</sender>
   <signal>clicked()</signal>
   <receiver>SymbolInfoDialogForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
